src/components/TodoItems.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import TodoForm from './TodoForm';
+
+// 할 일들을 관리(추가/삭제/토글)하는 컴포넌트
+const TodoItems = () => {
+  // items: 전체 할 일들을 저장
+  const [items, setItems] = useState([]);
+
+  // inputText: 입력한 할 일
+  const [inputText, setInputText] = useState('');
+
+  // 인풋 필드에 입력한 값으로 inputText를 변경
+  const handleInputChange = (e) => {
+    setInputText(e.target.value);
+  };
+
+  // items 배열에 입력한 할 일을 추가
+  const addNewTodo = useCallback(
+    (e) => {
+      e.preventDefault();
+
+      // 중복 검사를 위한 findIndex
+      const indexOfDuplicates = items.findIndex(
+        (todo) => todo.text === inputText
+      );
+
+      // 중복, 공백 입력이 아니면 객체 형태로 만들어 items에 저장
+      if (inputText && indexOfDuplicates === -1) {
+        const todoItem = {
+          text: inputText,
+          isDone: false,
+        };
+
+        setItems(items.concat(todoItem));
```

Comment:
```suggestion
        setItems([...items, todoItem]);
```

간지나게 `spread operator`를 써봅시다!

Reply:
간지나게 수정했습니다